Guides/Image_Formats.md

```diff
@@ -2,46 +2,44 @@
 
-It's also of note that the .IMG itself should be identical to the .BIN, as they're both essentially raw copies of the disc's data. It also only supports CDs as well, for the same reasons that CUE only supports CDs. This means that in practice, a CCD image can be thought of a more rigidly defined CUE image that also supports subchannels.
+### Media Descriptor Image (v1) (.MDS/.MDF)
 
-### Media Descriptor File (v1) (.MDS/.MDF)
+The "Media Descriptor Image" format, almost always referred to as the MDS format after its "Media Descriptor Sidecar" extension, is once again similar to CUE and CCD in its basic structure. The .MDS file is similar to the .CCD file in that it contains information about the disc's structure (along with additional data) which is *not* presented in a human readable format. The .MDF ("Media Descriptor File") file is also similar to the .BIN and .IMG files, except that it can also contain the subchannel data directly, without using a .SUB file. This has pros and cons: it makes for less file clutter, but subchannel data is essentially never consistently dumped, meaning that an .MDF file that contains this data will not be usable for reliable DAT'ing (where the exact file hash needs to be consistent between dumps).
 
-The MDS format is, once again, similar to CUE and CCD in the basic structure. The .MDS file is similar to the .CCD file in that it again contains information about the disc's structure (along with additional data), while *not* being presented in a human readable format. The .MDF file is also similar to the .BIN and .IMG files, except that it can also contain the subchannel data directly, without using a .SUB file. This has pros and cons, with the main pro being that there are less files present, decluttering things slightly. The main con is that subchannel data is essentially never consistently dumped, meaning that an .MDF file that contains this data will not be usable for reliable DAT'ing, where the exact file hash needs to be consistent between dumps. Otherwise, the content of the .MDF file is equivalent to the data present in the .BIN and .IMG files where applicable.
+The main reason to use MDS for CDs is the fact that it supports more devious copy protection schemes directly, most notably ones that make use of DPM, in which the disc is mastered to intentionally make certain sectors take non-standard amounts of time to read, and these timings are measured, stored within the .MDS file, and then used to verify the disc's authenticity. Only certain very specific virtual drives support this feature, including Alcohol 120% (the program this format was originally designed for) and Daemon Tools.
```

Comment:
No reason to specifically reference using MDS for CDs in this sentence, as MDS is generally most used for copy protected media as a whole, not just for CDs. Also, the first sentence is too much of a run-on sentence for my liking.